ostd/src/mm/stat/mod.rs

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! Statistics prepared for `/proc/meminfo`.
+
+use crate::mm::page::allocator::PAGE_ALLOCATOR;
+
+/// Total memory. For `MemTotal` in `/proc/meminfo`.
```

Comment:
```suggestion
/// Total memory available for any usage s in the system (in bytes).
///
/// It would be only a slightly less than total physical memory of the system in most occasions.
/// For example, bad memory, kernel statically-allocated memory or firmware reserved memories
/// does not count.
```